lib/AppInfo/Application.php

```diff
@@ -21,53 +21,45 @@
 
 namespace OCA\Notifications\AppInfo;
 
+use OCA\Notifications\App;
 use OCA\Notifications\Capabilities;
 use OCA\Notifications\Controller\EndpointController;
-use OCA\Notifications\Handler;
-use OCP\AppFramework\App;
-use OCP\IContainer;
-use OCP\IUser;
-use OCP\IUserSession;
+use OCP\Util;
 
-class Application extends App {
-	public function __construct (array $urlParams = array()) {
-		parent::__construct('notifications', $urlParams);
+class Application extends \OCP\AppFramework\App {
+	public function __construct() {
+		parent::__construct('notifications');
 		$container = $this->getContainer();
 
-		$container->registerService('EndpointController', function(IContainer $c) {
-			/** @var \OC\Server $server */
-			$server = $c->query('ServerContainer');
+		$container->registerAlias('EndpointController', EndpointController::class);
```

Comment:
But that is automagically resolved...  https://github.com/nextcloud/server/blob/master/lib/private/AppFramework/App.php#L114

So it will look for EndpointContoller first. And if it can't find it 'OCA\<APP>\Controller\EndPointcontoller'

Anyway it works